src/sentry/models/exporteddata.py

```diff
@@ -97,6 +97,19 @@ def email_success(self):
         )
         msg.send_async([self.user.email])
 
+    def email_failure(self, message):
+        from sentry.utils.email import MessageBuilder
+
+        msg = MessageBuilder(
+            subject="Unable to Export Data",
+            context={"error_message": message},
+            type="organization.export-data",
```

Comment:
Not sure what a good `type` description would be -- let me know if there's a convention

Reply:
@leeandher I don't think `type` is required. What happens if you omit it?

Reply:
It's not required, I think it's just a categorization thing for us.